test/Python/python_runtime.swift

```diff
@@ -6,17 +6,32 @@
 import Python
 import StdlibUnittest
 
+let gc = Python.import("gc")
+extension TestSuite {
+  func leakCheckTest(_ name: String, body: @escaping () -> ()) {
```

Comment:
```suggestion
  func testWithLeakCheck(_ name: String, body: @escaping () -> Void) {
```
Please also add a doc comment: ``/// Check that running `body` does not cause Python memory leaks.``